tools/SetupFlow/DevHome.SetupFlow/ViewModels/PackageViewModel.cs

```diff
@@ -203,22 +204,14 @@ private void ToggleSelection()
         _screenReaderService.Announce(announcementText);
     }
 
-    /// <summary>
-    /// Gets the package icon based on the provided theme
-    /// </summary>
-    /// <param name="theme">Package icon theme</param>
-    /// <returns>Package icon</returns>
-    private BitmapImage GetIconByTheme(RestoreApplicationIconTheme theme)
+    public BitmapImage GetLightThemeIcon()

```

Comment:
It's a silly difference but can we do 
```suggestion
    private BitmapImage GetIconByTheme(ElementTheme theme)
```
and just call with the right element theme? It's less changed. Also can this method stay private?